mvp.feature

```diff
@@ -193,13 +193,13 @@ Feature: User with no Author identity in the Authority Register for Personas see
     And they see that the Other Contributors have the correct Role, Name and Institution
```

Comment:
Hvordan ser man at `Authors are verified in the Authority Register for Personas`? 
Hva er `expected order`?
Hva betyr det at `the correct authors have the expected value for corresponding author`?